<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditComputer</class>
 <widget class="QDialog" name="EditComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Computer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/images/genius-icon.png</normaloff>:/icons/images/genius-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditComputer{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(185, 185, 185, 255), stop:1 rgba(225, 225, 225, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;h1&gt;Update Computer</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelModelName">
       <property name="text">
        <string>Model Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelModelNameError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditModelName"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelMakeYear">
       <property name="text">
        <string>Make Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMakeYearError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMakeYear"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTypeError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxType"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBuilt">
     <property name="text">
      <string>Built</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="buttonGeniusRelationship">
     <property name="styleSheet">
      <string notr="true">#buttonGeniusRelationship {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(33, 33, 113, 255), stop:1 rgba(136, 175, 255, 255));
	color: rgb(255, 255 ,255);
	padding: 6px 15px;
	border: 2px solid;
	border-color: transparent;
	border-radius: 14px;
}

#buttonGeniusRelationship::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(63 63, 113, 255), stop:1 rgba(136, 175, 255, 255));
}</string>
     </property>
     <property name="text">
      <string>Edit genius relationship</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="styleSheet">
      <string notr="true">#pushButtonSave{
	color: rgb(255, 255, 255);
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(31, 107, 31, 255), stop:1 rgba(110, 189, 100, 255));
	padding: 6px 15px;
	border: 2px solid;
	border-color: transparent;
	border-radius: 15px;
}

#pushButtonSave::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(10, 35, 10, 255), stop:1 rgba(110, 189, 100, 255));
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
